### ISSUE TITLE
Too many Helper processes ?

### ISSUE THREAD

**Issue by smallfly** [ofZach/ofxCef#26]
I just ran the example and realized that there is now 2 Helper processes for the same application. One named as the application 'example-ofxCEF Helper' and one named 'cefHelper'.

![Activity Monitor](http://smallfly.com/of/ofxCEF-HelperProcesses.png)

**Comment by ofZach** [comment]
yeah the helper process stuff is kind of insane.  as far as I know, there are actually quite a few helpers (see "make_more_helpers.sh").   My understanding that is cefHelper gets changed to have the app name + "Helper" and then copied / name changed to have a bunch of different settings applied, but I see for example three helpers in the last script that's applied on compile:

```
exec install_name_tool -change "./libfmodex.dylib" "@executable_path/../../../../MacOS/libfmodex.dylib"  "$TARGET_BUILD_DIR/$PRODUCT_NAME.app/Contents/Frameworks/$PRODUCT_NAME Helper.app/Contents/MacOS/$PRODUCT_NAME Helper"
exec install_name_tool -change "./libfmodex.dylib" "@executable_path/../../../../MacOS/libfmodex.dylib"  "$TARGET_BUILD_DIR/$PRODUCT_NAME.app/Contents/Frameworks/$PRODUCT_NAME Helper.app/Contents/MacOS/$PRODUCT_NAME Helper EH"
exec install_name_tool -change "./libfmodex.dylib" "@executable_path/../../../../MacOS/libfmodex.dylib"  "$TARGET_BUILD_DIR/$PRODUCT_NAME.app/Contents/Frameworks/$PRODUCT_NAME Helper.app/Contents/MacOS/$PRODUCT_NAME Helper NP"
```

I'm not 100% clear what's happening, but how does this compare to running a more traditional OSX cef example?

**Comment by smallfly** [comment]
The applications 'cefclient' and 'cefsimple' included in the package 'cef_binary_3.2295.2034_macosx64' show the following in the Activity Monitor.

![Helper Processes](http://smallfly.com/of/ofxCEF-HelperProcesses-02.png)
